### DISCUSSION TITLE
健康真的很重要

### DISCUSSION THREAD

**Discussion by christaaah** [1101-GS4719-C/Python-Community-GS4719-C#2254] [General]
今天華新聞的時候滑到一篇標題為《安寧緩和照護護士揭露臨終病人最常見的後悔》的報導[https://www.msn.com/zh-tw/news/other/%E5%AE%89%E5%AF%A7%E7%B7%A9%E5%92%8C%E7%85%A7%E8%AD%B7%E8%AD%B7%E5%A3%AB%E6%8F%AD%E9%9C%B2%E8%87%A8%E7%B5%82%E7%97%85%E4%BA%BA%E6%9C%80%E5%B8%B8%E8%A6%8B%E7%9A%84%E5%BE%8C%E6%82%94/ar-AA1tHFnG?ocid=msedgdhp&pc=U531&cvid=44818730876b4c62a7c89f1e275b3bb1&ei=24](url)
裡面的護士提到那些臨終的病人最後悔的就是沒有在自己健康的時候好好注意自己的健康，所以她每天都會提醒自己注意健康。因為最近家人也生病，因為平時沒有表現出不舒服，所以也沒有多注意，直到有一次比較嚴重才發現，其實身體和心理已經很勞累了。也因為這樣，讓我覺得健康真的非常重要，也要多注意家中父母、老人的健康，能叫他們去做體檢就叫。

**Comment by Chattenoire** [comment]
面對臨終，是大多數人終究會碰到的問題；如果想要透過文學了解可能的情境與情緒，推薦看看《一場極為安詳的死亡》或是安妮艾諾的《位置》，兩本都是很深刻的書

**Comment by wcyeee** [comment]
我也覺得健康很重要！但好像大家都知道這件事，卻還是很多人不太在意健康，要等到真的失去了什麼還是自己感覺到了，才會真正注重健康這件事。所以我覺得大家應該要平常就好好注意自己的身體，定期健檢運動，好好吃飯睡覺，不要等到發現不對勁了才開始注意這些簡單的事情。

**Comment by forestdew507** [comment]
健康這件事的重要性我是在暑假服務了長者後才突然有的感悟...... 從年輕的時候就把自己練強壯，以後老了想去哪還可以自己去，體體面面、舒舒服服，可以繼續完成自己人生清單上的事，難道不好嗎 !

**Comment by ShuliANN** [comment]
我也認為健康非常重要，尤其是我們年輕人，往往容易忽視飲食和睡眠的重要性。不少人習慣吃垃圾食物，對健康飲食漠不關心，甚至仗著年輕常常熬夜，卻不意識到這些行為會對健康造成長期影響。大家都要提醒自己和身邊的人，一起保持身心健康！

**Comment by yue083139** [comment]
我的爺爺奶奶是還蠻自律的人，早睡早起注重飲食，即便如此最近也到了會需要開大大小小手術的年紀了，不禁讓我覺得說如果從現在就不節制、注意自己身體的話，未來可能要面臨到更加嚴重的生理問題，因此我最近都會告誡自己不要熬夜到太晚睡。

**Comment by yuan3x2** [comment]
我也覺得健康很重要，不管是生理上還是心理上的健康，要先擁有一個健康的身心靈，才能夠好好享受生活所帶來的快樂，我最近也開始感受到健康的重要性了，希望大家都有好好睡覺、多運動、吃得營養、適度讓眼睛休息，不然等年齡優勢過去了，就會開始後悔自己沒有好好對待自己的身體。

**Comment by lai1008** [comment]
我也覺得健康很重要！雖然大家都知道這件事，卻還是很多人直到身體出現狀況才開始重視。因此我覺得我們平常就應該好好注意自己的身體，養成運動的習慣，好好吃飯睡覺，不要等到來不及了才開始改變自己
。

**Comment by Mminniehuang** [comment]
我之前在運動場館工讀的時候，總是會遇到很多阿公阿嬤每天早起運動，有一個伯伯甚至說他三十幾年來每天早上都堅持到中央運動，勸我們一定要養成良好運動習慣。
真的很多人看起來都比我們年輕人還健壯，所以我們真的要好好反思，不要隨意揮霍掉自己的健康之後再後悔...

**Comment by Elaine4488** [comment]
健康真的很重要！每一次都等到身體出狀況才會後悔為什麼作息不正常。每天都希望自己能夠早睡早起，但永遠做不到XD
